flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -1,27 +1,37 @@
 package flixel.system.frontEnds;
 
 import flixel.math.FlxPoint;
+import flixel.util.FlxStringUtil;
 
 #if js
 import js.Browser;
 
 class HTML5FrontEnd
 {
-	public var browser(get, never):FlxBrowser;
-	public var browserWidth(get, never):Int;
-	public var browserHeight(get, never):Int;
+	public var browser(default, null):FlxBrowser;
+	public var browserWidth(default, null):Int;
+	public var browserHeight(default, null):Int;
 	public var browserPosition(get, null):FlxPoint;
+	public var platform(default, null):FlxPlatform;
+	public var isMobile(default, null):Bool;
 	
 	@:allow(flixel.FlxG)
-	private function new() {}
+	private function new()
+	{
+		browser = getBrowser();
+		browserWidth = getBrowserWidth();
```

Comment:
Browser width and height could change at runtime, so I don't think caching those is a good idea.